bin/dlang/install

```diff
@@ -14,7 +14,7 @@ case "$DMD" in
     dmd*   ) PKG= ;;
     1.*    ) PKG="dmd1=$DMD-$DIST" ;;
     2.*.s* ) PKG="dmd-transitional=$DMD-$DIST" ;;
-    2.*    ) PKG="dmd-bin=$DMD libphobos2-dev=$DMD" ;;
+    2.*    ) PKG="dmd-compiler=$DMD dmd-tools=$DMD libphobos2-dev=$DMD" ;;
```

Comment:
If old packages are still named `dmd-bin`, shouldn't we check the exact version to get and if `test $(cut -d. -f2 $DMD) ge 77` only then we use the new package name?